bin/validate-policies.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -uo pipefail
+
+EXIT_FAST=${EXIT_FAST:-"1"}
+[[ $EXIT_FAST == "1" ]] && set -e
+
+k8sResourcesPath="/tmp/otomi/conftest-fixtures"
```

Comment:
can we use `mktemp -u` here as well?

Reply:
We can't use tmp folder here. When using MNT_TMP_DIR, removing this folder will be ignored and we don't want random tmp folder cluttering on local workstations.